README.md

```diff
@@ -52,22 +52,27 @@ Potential drawbacks:
   ballpark of around 15%.
 * API breakage. If you can't afford modifying your code to adhere to the new API
   and don't care too much about the new features, v2 is probably not for you.
-* Some bug fixes are trailing behind and still need to be forward-ported to v2.
-  See issue #348 for tracking.
+* Several bug fixes are trailing behind and still need to be forward-ported to
+  v2. See issue [#348](https://github.com/russross/blackfriday/issues/348) for
+  tracking.
 
 Usage
 -----
 
 For the most sensible markdown processing, it is as simple as getting your input
 into a byte slice and calling:
 
+``` go
     output := blackfriday.Run(input)
+```
```

Comment:
When you go from indented code block to fenced code block, you should get rid of the indentation, since it'll now be a part of the code block. This should become:

    ```go
    output := blackfriday.Run(input)
    ```

Also, it's more common not to include a space after the three backticks.